src/components/ProfilePage/index.jsx

```diff
@@ -0,0 +1,67 @@
+import React, { useState, useEffect, useContext } from "react";
+import GamesCarousel from "../GamesCarousel";
+import { useTranslation } from "react-i18next";
+import "./index.css";
+import { Col, Layout } from "antd";
+import profilePageImg from "../../images/profilePage/profilePageImg.png";
+import db from "../../firebaseConfig";
+import { v4 as uuidv4 } from "uuid";
+import { AuthContext } from "../../auth/authContext";
+
+const { Sider, Content } = Layout;
+const ProfilePage = () => {
+  const { t } = useTranslation();
+  const { currentUser } = useContext(AuthContext);
+  const currentUserId = currentUser ? currentUser.uid : null;
+  const [user, setUser] = useState([]);
```

Comment:
* how many users is this profile page for?
* when you use an array, how many things can an array store?
* does it make sense to use an array when there is only one user?

Reply:
@louisrli  sorry fixed it